Source/ACE.Entity/Vector3.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.IO;
+using System.Numerics;
+
+namespace ACE.Entity
+{
+    public class AceVector3
```

Comment:
Will add ability to get a real vector 3 but it's not what this was intended for but it could be added as a way to retrieve it through a protected function